### ISSUE TITLE
RSA signature example is SHA-256 or SHA-512?

### ISSUE THREAD

**Issue by chrisinmtown** [cryptoexamples/python-cryptography-cryptoexamples#3]
Thank you very much for these examples. I would like to ask a question about https://github.com/cryptoexamples/python-cryptography-cryptoexamples/blob/master/src/cryptoexamples/example_string_signature_rsa.py

The comment at the top says "SHA-512 with RSA signature of text using PSS and MGF1 padding" but in the code I only see invocations of SHA256 like this:
```
            padding=padding.PSS(
                mgf=padding.MGF1(hashes.SHA256()),
                salt_length=padding.PSS.MAX_LENGTH
            ),
            algorithm=hashes.SHA256()
```
Please forgive my crypto ignorance and tell me, is this just a case of a comment not matching the code, or is there much more going on that I need to understand?  I'm asking because I need to write a requirement to compute a digital signature without supplying python code, and I'm trying to grasp what I need to say about hashes etc.

**Comment by kmindi** [comment]
@chrisinmtown thanks for your comment. You are totally right, this is indeed a case of a comment not matching the code. I'll create a PR to align this.

**Comment by kmindi** [comment]
closed by 77bef3b5d1532ea198ff2db4055c78d1ee2b2ee7
